src/components/Event/Event.tsx

```diff
@@ -26,40 +35,76 @@ export class Event extends React.Component<EventProps> {
     showTime: true,
     fullday: false,
     selected: false,
+    disabled: false,
+    onClick: null,
+    'aria-has-popup': false,
+    'aria-expanded': false,
   };
 
   getDataAttributes() {
-    const { fullday, selected, showTime } = this.props;
+    const { fullday, selected, showTime, onClick, disabled } = this.props;
 
     return {
       [EVENT_DATA_KEYS.IsFullDay]: fullday,
       [EVENT_DATA_KEYS.IsSelected]: selected,
       [EVENT_DATA_KEYS.IsTimeShown]: showTime,
+      [EVENT_DATA_KEYS.OnClick]: Boolean(onClick),
+      [EVENT_DATA_KEYS.Disabled]: disabled,
```

Comment:
I think that in these cases we don't need data- attributes, because we can actually read them straight from the dom